README.md

```diff
@@ -2,7 +2,27 @@
 
 These models are used to communicate with the Apps Rendering API
 
-## How to release
+## How to release to NPM
+
+This repository uses [`changesets`](https://github.com/changesets/changesets) to manage versions and releases.
+
+To create a changeset, ensure you are using the correct Node version defined in [`.nvmrc`](./.nvmrc) and project dependencies installed by running `npm install`. Then:
```

Comment:
Should we mention that people should also use the `npm` version that corresponds to this Node version?